catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -384,7 +464,7 @@ public void testExecuteQueryCacheDisabled() throws Exception {
         searchController.executeQuery(request, mockServerSession, null);
 
         // Verify
-        verify(framework).query(queryRequestCaptor.capture());
+        verify(framework, times(2)).query(queryRequestCaptor.capture());
```

Comment:
✏️  It would help readability to extract the value 2 and set as a final constant at the top of the class.